src/search.ts

```diff
@@ -3,6 +3,9 @@ import {CallExpression, Node} from 'estree';
 
 const walk = require('acorn/dist/walk');
 
+export const iOModules: string[] =
```

Comment:
For naming purposes let's treat acronyms like normal words. (I'm calling I/O an acronym) So it would be `ioModules`.

Reply:
done